requirements.txt

```diff
@@ -1,5 +1,6 @@
-pyhive[hive_pure_sasl]~=0.7.0
-requests>=2.28.1
+PyHive[hive]>=0.6.0,<0.8.0
```

Comment:
@colin-rogers-dbt I want to get your thoughts on this. I seem to remember you did something with moving to `hive_pure_sasl` as a way to support py311. Is that right? Put another way, does this look right? Or should we stick with the existing dependencies and close this PR?